setup/symfony_server.rst

```diff
@@ -0,0 +1,271 @@
+Symfony Local Web Server
+========================
+
+You can run Symfony applications with any web server (Apache, nginx, the
+internal PHP web server, etc.) However, Symfony provides its own web server to
+make you more productive while developing your applications.
+
+Although this server is not intended for production use, it supports HTTP/2, TLS/SSL,
```

Comment:
We should make this more prominent, that it is not for production use, with a `.. caution::` directive, WDYT @javiereguiluz ?

Reply:
I don't think it's needed ... and also the article title itself calls it "local web server".